cookiecutter/cli.py

```diff
@@ -108,6 +113,13 @@ def main(
         debug_file=debug_file,
     )
 
+    # If needed, prompt the user to ask whether or not they want to execute
+    # the pre/post hooks.
+    if accept_hooks == 'ask':
+        ignore_hooks = not click.confirm('Do you want to execute hooks?')
+    else:
+        ignore_hooks = accept_hooks == 'no'
+
```

Comment:
Can we please switch this up so that the variable is also `accept_hooks`?
This will let you make [L294](https://github.com/audreyr/cookiecutter/pull/992/files#diff-b4c1da8a76ea242bf99a397dcb90b2a6R294) more natural sounding?

Reply:
Addressed.